pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -447,6 +464,14 @@ func aclToRules(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) (types.IPTabl
 			aclRule4.Action = []string{"-j", "ACCEPT"}
 			rulesList = append(rulesList, aclRule1, aclRule2, aclRule3, aclRule4)
 
+			aclRule1.Rule = []string{"-i", aclArgs.BridgeName, "-d", "169.254.169.254",
+				"-p", "tcp", "--dport", "http"}
+			aclRule1.Action = []string{"-j", "ACCEPT"}
+			aclRule2.Rule = []string{"-i", aclArgs.BridgeName, "-s", "169.254.169.254",
```

Comment:
Was the intention to use "-o" here? With "-o" rule is prefixed to add into filter FORWARD chain. And rules in the chain will not match packets in this case. I think, your case will work even with out this particular rule.

Reply:
It is copy and paste from the tcp domain rules which use -i.

But as I said, I don't know if we have redundant rules for dhcp and dns which we should clean up.

Reply:
It is possible that the original rules where your copied from are wrong. Don't know if these is a case where these rules can match. If not, we should probably remove these.